packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -302,8 +308,14 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
 }
 
 class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>> {
-  final GlobalKey _fieldKey = GlobalKey();
   final LayerLink _optionsLayerLink = LayerLink();
+  final GlobalKey _fieldKey = GlobalKey(debugLabel: kReleaseMode ? null : 'AutocompleteFieldView');
+
+  // TODO(justinmc): Maybe no longer needed?
```

Comment:
Should this comment be removed?

Reply:
It looks like the value of _fieldBoxConstraints is never read, but maybe it's still needed to trigger a rebuild in the options view when the field layout changes? Can you try removing it and see if it breaks anything?